lib/src/base-styles/_nx-colors.scss

```diff
@@ -17,8 +17,8 @@
 
   --nx-color-interactive-background-selected: var(--nx-swatch-blue-90);
   --nx-color-interactive-background-active: var(--nx-swatch-blue-95);
-  --nx-color-interactive-background-hover: var(--nx-swatch-indigo-95);
-  --nx-color-interactive-background-disabled: var(--nx-swatch-grey-95);
+  --nx-color-interactive-background-hover: var(--nx-swatch-grey-90);
```

Comment:
tables, lists and dropdowns are the only components that use this variable, and all have `--nx-swatch-grey-90` as their background hover colour according to updated Kaleido mockups, so updated here